docker/local/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.2"
+services:
+ redis:
+    image: "redis:alpine"
+
+    command: redis-server
+    expose:
+     - "6379"
+
+    volumes:
+     - $PWD/redis-data:/var/lib/redis
```

Comment:
Maybe we can add a comment about this directory (and how it can be changed) in the README?

Reply:
ack, done